.github/workflows/build_wasm.yml

```diff
@@ -0,0 +1,66 @@
+name: build_wasm
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+jobs:
+
+  build_linux:
+    name: Build_Linux
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: Linux_EMCC_2
+          os: ubuntu-20.04
+          compiler_version: "2.0.13"
+
+    steps:
+    - name: Sync Repository
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - uses: actions/checkout@v2
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+         node-version: '12'
+
+    - name: Install Dependencies (Linux)
+      run: |
+        sudo apt-get update
+        sudo apt-get install libglu1-mesa-dev xorg-dev
+        mkdir -p tmp
+        cd tmp
+        git clone --recursive https://github.com/emscripten-core/emsdk
+        cd emsdk
+        ./emsdk install ${{ matrix.compiler_version }}
+        ./emsdk activate ${{ matrix.compiler_version }}
+        source ./emsdk_env.sh
+        echo "EMSDK=$EMSDK" >> $GITHUB_ENV
+
+    - name: Create Build Directory
+      run: mkdir build
+      
+    - name: CMake Generate
+      run: cmake -DMATERIALX_BUILD_JS=ON -DMATERIALX_EMSDK_PATH=${{ env.EMSDK }} -DMATERIALX_BUILD_RENDER=OFF -DMATERIALX_BUILD_TESTS=OFF -DMATERIALX_TEST_RENDER=OFF -DMATERIALX_WARNINGS_AS_ERRORS=ON ${{matrix.cmake_config}} ..
+      working-directory: build
+
+    - name: CMake Build
+      run: cmake --build . --config Release --parallel 2
+      working-directory: build
+    
```

Comment:
Will the install step be added as part of this PR or later on ? We can always do it afterwards. Once we do the install step then can run an action to copy the artifact back to this area as we do with other workflows.

Reply:
Ah ok, I didn't know that was the purpose. I explicitly removed the install since emscripten generates a file that doesn't need to be added to an include path or anything. What exactly do you do with the artifacts? I would suggest start with this covering the CI and then include the CD part

Reply:
Yes it should pass without installing :).

At some point we will be posting installs on our github and the ILM one.
Thus we also include this a "packaging" build step. Something like this (taken from build_linux.yml -- assume build/installed is the install location).
```
 - name: Upload Installed Package
      uses: actions/upload-artifact@v2
      with:
        name: MaterialX_${{ matrix.name }}
        path: build/installed/
```

Reply:
True! sorry, I didnt see that.